app/api/v1/auth/index.js

```diff
@@ -175,4 +212,25 @@ router.post('/verification', (req, res, next) => {
   });
 });
 
+/**
+ * Verifies a user's email based on a code sent out to the user's provided email
+ * Responds with a JSON object with fields 'error', 'verified' (if a user is succesfully verified)
+ */
+router.post('/verifyEmail', (req, res, next) => {
+  if (!req.body.code || !req.body.email) {
+    return next(new error.BadRequest('Email and Email verification code must be provided'));
+  }
+  User.findByEmail(req.body.email.toLowerCase()).then(async (user) => {
+    if (!user) return res.json({ error: 'Invalid user email', verified: false });
```

Comment:
The error handler middleware takes care of status codes and error messages, so throw an error instead of responding.